### DISCUSSION TITLE
Computer Locked

### DISCUSSION THREAD

**Discussion by StormyIceLeopard** [StreamController/StreamController#94] [General]
Hello
I'm unsure how this works right now.
My setup is packed away for moving
Hoping to be settled in a month or less.
But I'm wondering if this has been addressed on this software
https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/issues/23

**Comment by Core447** [comment]
No, I still need to find a good way to detect this on all environments.

**Comment by Core447** [comment]
`1.5.0-beta.4` added auto-lock for Gnome. It will be available on Flathub in the next couple of hours

**Reply by Caralluin** [reply]
Is this controllable by configuration? In some circumstances, leaving the deck active may even be exactly what you want, e.g. if you want to control your media player even if the computer is locked. (I do not use Gnome, so I don't expect to see any difference soon).

**Reply by Core447** [reply]
> Is this controllable by configuration? In some circumstances, leaving the deck active may even be exactly what you want, e.g. if you want to control your media player even if the computer is locked. (I do not use Gnome, so I don't expect to see any difference soon).

For now there's no way to disable this behavior, but I'm going to implement a setting for it soon.
